projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.html

```diff
@@ -0,0 +1,16 @@
+
+<div tabindex="0" role = "button" [attr.aria-disabled] = "disabled" [attr.aria-expanded] = "isExpanded" [attr.aria-controls] = "controls">
+<ng-template #iconContainer>
+    <ng-content *ngIf="iconTemplate" select="igx-expansion-panel-icon"></ng-content>
+    <igx-icon *ngIf="!iconTemplate" fontSet="material" [name]="panel.collapsed? 'expand_more':'expand_less'"></igx-icon>
```

Comment:
We could deprecate the name input property on the IgxIcon component. Avoid using it for setting ligature icons. Instead specify the name between the opening and closing tags of the igx-icon selector.